commercetools-models/src/test/java/io/sphere/sdk/subscriptions/SubscriptionFixtures.java

```diff
@@ -69,6 +71,25 @@ public static void assumeHasAzureSBEnv() {
         assumeNotNull(azureSBConnectionString);
     }
 
+    /**
+     * Check if azure url is defined
+     */
+    public static void assumeHasAzureFunctionUrl(){
+        final String url = azureFunctionUrl();
+        assumeNotNull(url);
+    }
+
+
+    /**
+     *
+     * @return The azure function used to validate the resource lifecycle (create, delete ....)
+     * @see io.sphere.sdk.extensions.Extension
+     */
+    public static String azureFunctionUrl(){
+        final String ironMqUriEnv = System.getenv(AZUREFUNCTION_URL);
```

Comment:
Do we then add the `AZUREFUNCTION_URL` environment variable to our inhouse TeamCity build server?

Reply:
@katmatt yes, this was added to teamcity.